vnext/PropertySheets/NuGet.Cpp.props

```
@@ -16,7 +16,7 @@
     <ResolveNuGetPackages>false</ResolveNuGetPackages>
 
     <!-- https://github.com/NuGet/Home/issues/10511#issuecomment-778400668 -->
-    <AssetTargetFallback>$(AssetTargetFallback);uap10.0.16299</AssetTargetFallback>
+    <AssetTargetFallback>$(AssetTargetFallback);uap10.0.17763.0</AssetTargetFallback>
```

You might need to remove the `.0` for this to work properly

```suggestion
    <AssetTargetFallback>$(AssetTargetFallback);uap10.0.17763</AssetTargetFallback>
```

---

IIRC it's more of an identifier than a pure version.